.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+    "env": {
+      "browser": true,
+      "es2021": true,
+      "node": true
+    },
+    "extends": "eslint:recommended",
+    "overrides": [
+        {
+            "files": ["**/*.test.js"],
+            "env": { "jest": true },
+            "plugins": ["jest"],
+            "extends": ["plugin:jest/recommended"],
+            "rules": { "jest/prefer-expect-assertions": "off" }
+          },
+          {
+            "files": ["**/*.cy.js", "cypress.config.js"],
+            "env": { "cypress/globals": true },
+            "plugins": ["cypress"],
+            "extends": ["plugin:cypress/recommended"],
+            "rules": {
+              "cypress/no-unnecessary-waiting": "off",
+              "no-unused-vars": "off"
+            }
+          }
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
+        "cypress/no-unnecessary-waiting": "off",
+        "no-unused-vars": "warn"
+    }
+}
```

Comment:
Hi @openai, In this case it is necessary because we are handling unit test files in node. Jest uses undefined functions therefore this setting is required. Do you know of an alternative workaround to this issue?